components/BottomNavigation/tests/unit/BottomNavigationItemViewTests.m

```diff
@@ -127,6 +127,28 @@ - (void)testSetSelectedItemTintColorUpdatesInkColor {
   XCTAssertNotEqualObjects(item2.inkView.inkColor, item2DefaultInkColor);
 }
 
+- (void)testSetTitleVisibilityUpdatesLayout {
+  // Given
+  MDCBottomNavigationItemView *view = [[MDCBottomNavigationItemView alloc] init];
+  view.title = @"Test Content";
+  view.image = fakeImage();
+  view.bounds = CGRectMake(0, 0, 100, 100);
+  view.contentVerticalMargin = 20;
+  view.contentHorizontalMargin = 20;
+  view.titleVisibility = MDCBottomNavigationBarTitleVisibilityAlways;
+  [view layoutIfNeeded];
+  CGFloat imageMidY = CGRectGetMidY(view.iconImageView.frame);
+  XCTAssertEqualWithAccuracy(imageMidY, 32.75, 0.001);
```

Comment:
Tip: Consider making this a less-than-equal and greater-than-equal so you can verify the "approximate" location.  It seems like we don't care too much about the exact placement, but rather only that it is "to the left of center" by some reasonable amount.

Reply:
Good point. I appreciate you giving a broader view; I admit I was happy just to get a passing test that wouldn't have passed before.